pyschema/core.py

```diff
@@ -72,6 +72,20 @@ class ParseError(Exception):
     pass
 
 
+class InvalidSchemaSpecification(object):
+    """
+    Utility class that can be used to raise an exception on schema usage.
+
+    This is used in the schema store as a placeholder for invalid schemas. Instead of raising when a schema is
+    registered in the store, something that happens on import, we use this class to raise on usage.
+    """
+    def __init__(self, exception_to_raise):
```

Comment:
Either this parameter should be called exception_msg or the passed value should be an exception and not a string :)